paddle/gserver/layers/MKLDNNLayer.h

```diff
@@ -248,6 +266,77 @@ class MKLDNNLayer : public Layer {
   }
 
   /**
+   * reset the output grad matrix from primitive desc.
+   * and reset the merge grad primitive if needed.
+   * note: when this layer have serval output,
+   *       do not support mixing with cpu device,
+   *       because can not get memory desc from cpu device.
```

Comment:
when this layer has serval outputs, it could not be mixed with cpu device, since it can not get memory desc from cpu device.